### PR TITLE
VIMC-4550: Add changelog to report run body

### PR DESCRIPTION
**Pull request by r-ash**
This PR will:
* Add changelog to body of run report and pass through to report runner

Expects changelog to passed in as e.g.
```
{
  "type": "internal",
  "message": "my message"
}
```

### CODE REVIEW

File: inst/schema/RunRequest.schema.json
Comment (richfitz) [inline inst/schema/RunRequest.schema.json:1]: I believe that at present we allow multiple changelog entries to be passed to orderly_run via message; we might expand this to be an array of changelog entries?

File: inst/schema/spec.md
Comment (richfitz) [inline inst/schema/spec.md:20]:
```suggestion
  "changelog": [
    {
      "type": "internal",
      "message": "Added new output plot"
    }
  ]
```

File: tests/testthat/test-runner.R
Comment (richfitz) [inline tests/testthat/test-runner.R:726]: no longer have appveyor

**Review by richfitz — CHANGES_REQUESTED** [review]

**Review by EmmaLRussell — COMMENTED** [review]
This looks good to me - please don't merge it until the corresponding OW ticket is ready to go! I'll start looking at it now. https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-2161

File: inst/schema/RunRequest.schema.json
Comment (r-ash) [reply inst/schema/RunRequest.schema.json:1]: After discussing, orderly CLI not supporting multiple changelog messages at the moment so let's continue with 1 for now. If science ask for multiple we can add that as an enhancement in the future. https://mrc-ide.myjetbrains.com/youtrack/issue/VIMC-4600

File: tests/testthat/test-runner.R
Comment (r-ash) [reply tests/testthat/test-runner.R]: Thanks, I've removed all calls to this

**Comment by EmmaLRussell** [comment]
The frontend changes for supplying changelog are ready now, so you can go ahead and merge this when you like, then I'll repoint the OW branch to master and merge.

**Pull request merged**
